tests/artifacts/openebs-ci.yaml

```diff
@@ -36,7 +36,7 @@ rules:
   resources: ["customresourcedefinitions"]
   verbs: [ "get", "list", "create", "update", "delete"]
 - apiGroups: ["*"]
-  resources: [ "disks"]
+  resources: [ "disks", "blockdevices", "blockdeviceclaims"]
```

Comment:
I am not able to understand the motivation behind this PR.
Can we update the PR message?